lib/model/narrow.dart

```diff
@@ -112,7 +112,7 @@ class TopicNarrow extends Narrow implements SendableNarrow {
   bool containsMessage(MessageBase message) {
     final conversation = message.conversation;
     return conversation is StreamConversation
-      && conversation.streamId == streamId && conversation.topic == topic;
+      && conversation.streamId == streamId && conversation.topic.isSameAs(topic);
```

Comment:
Hmm, good catch.

    narrow: Use TopicName.isSameAs in TopicNarrow.containsMessage
    
    TopicNarrow.containsMessage used to use raw equality operator (==)
    for comparing topics, which was not ideal for how we treat two
    topics the same.

Did this have a user-visible symptom? What were the symptoms?